roles/openshift_health_checker/library/search_journalctl.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/python
+"""Interface to journalctl"""
+
+from time import time
+import json
+import re
+import subprocess
+
+from ansible.module_utils.basic import AnsibleModule
+
+
+class InvalidMatcherRegexpException(Exception):
```

Comment:
Is there a need to have `Exception` in the name? One of @rhcarvalho 's style videos mentioned naming exceptions better. Couldn't it just be `InvalidMatcherRegexp`? And below, `InvalidMatcher`?